CMakeLists.txt

```diff
@@ -340,6 +340,8 @@ if(EMSCRIPTEN)
     # Build the .html (which builds the .js, .wasm, and .data too).
     set_target_properties(openttd PROPERTIES SUFFIX ".html")
     target_link_libraries(openttd WASM::WASM)
+    file(COPY ${CMAKE_SOURCE_DIR}/os/emscripten/demo.sav
```

Comment:
Don't mind this line. I wanted to include a demo save file into the browser version of the game. So ppl can try the functionality if any question.
Before merging I will revert this change.

Reply:
It applies to files `CMakeLists.txt` and `os/emscripten/demo.sav`